docs/news.md

```diff
@@ -4,6 +4,31 @@ summary: Important events happening.
 ---
 # News
 
+### 2021-01-31 - ASLR enabled by default
+
+**Backstory:** About 5 months ago we started backporting patches to our binutils
+2.35 to allow enabling ASLR support via various flags. We also enabled these
+flags in our build system, so any package in our repo that was updated in the
+last 5 months has ASLR support enabled. We've now updated to 2.36 which in
+theory has ASLR enabled by default but have reverted the defaults for now to
+make debugging easier in case of regressions.
+
+**Next steps:** In the coming days we will enable ASLR for everyone by default.
```

Comment:
That should be "for every packages by default". I was not built with ASLR 🤣

Reply:
yeah yeah :laughing:

Reply:
you're missing out ;)